### PR TITLE
Init values

### PR DESCRIPTION
**Pull request by Basir-Mohammadi**
## Project requirements

- Create class Person with the following:
- Instance vars: @id, @name, and @age.
- Constructor with name, age, and parent_permission as parameter. name and parent_permission are optional and have default values of "Unknown" and true.
- Getters for @id, @name, and @age.
- Setters for @name and @age.
- Private method is_of_age? that returns true if @age is greater or equal to 18 and false otherwise.
- Public method can_use_services? that returns true if person is of age or if they have permission from parents.
- Create class Student with the following:
- Inherits from Person.
- Constructor extends parent's constructor by adding @classroom and a parameter for it.
- Method play_hooky that returns "¯\(ツ)/¯".
- Create class Teacher with the following:
- Inherits from Person.
- Constructor extends parent's constructor by adding @specialization and a parameter for it.
- Override can_use_services? so it always returns true.
- Each class should be saved in a separate file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,46 @@
...
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Fellow Micronauts
+
+## 📝 License
+
+This project is MIT licensed.
```
Comment (OLIPLICHE) [inline README.md:46]: ## Optional
- Please fix `MIT` link. you can refer to the ReadMe template provided by Microverse

File: student.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+require './person'
+
+class Student < Person
+  def initialize(age:, name:, classroom:, parent_permission: true)
+    super(age: age, name: name, parent_permission: parent_permission)
+
+    self.classroom = (classroom)
```
Comment (OLIPLICHE) [inline student.rb]: Please Constructor extends parent's constructor by adding `@classroom and a parameter` for it: 
**Hint**: @classrom = classroom
Kindly  Open this Link 👉 [constructor-and-instance-vars](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars) to Learn More

File: teacher.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(age:, name:, specialization:, parent_permission: true)
```
Comment (OLIPLICHE) [inline teacher.rb]: - Please change `def initialize(age:, name:, specialization:, parent_permission: true)` to `def initialize(age, specialization, name = "Unknown", parent_permission = true)`. Remove the colons `(:)`
- Kindly Update the rest of the class above ( student, person).
- Use this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars) to learn more

File: person.rb [hunk 4]
```diff
@@ -0,0 +1,22 @@
+class Person
+  attr_reader :id
+  attr_accessor :age, :name
```
Comment (OLIPLICHE) [inline person.rb:3]: Congratulations on using the shortcut Method.
Link to learn more 👉 [Shortcut](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#useful-shortcuts-attr_reader-attr_writer-and-attr_accessor)

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @Basir-Mohammadi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linter is green (the last commit have a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏
- Gitflow is correctly used 👏

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sumairq — APPROVED** [review]
### Hi @Basir-Mohammadi  , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Git flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**